core/modules/system/system.install

```diff
@@ -568,7 +568,20 @@ function system_requirements($phase) {
       'description' => $description,
     );
   }
-
+  
+  if ($phase == 'runtime') {
+    // Inform user if theme_debug is enabled.
+    $theme_debug_enabled = config_get('system.core', 'theme_debug');
+    if ($theme_debug_enabled) {
+      $requirements['theme_debug'] = array(
+        'title' => $t('Theme debug'),
+        'value' => $t('Enabled'),
+        'severity' => REQUIREMENT_WARNING,
+        'description' => $t('Theme debug is enabled, probably via "Devel" module. Do not forget to disable this module when you have finished testing and this site goes live.'),
```

Comment:
> Theme debug is enabled, probably via "Devel" module. 

I'm not sure I like this language. "Theme debug" is a strange noun to put in the user interface. Can we say "Theme debugging" here instead?